cmd/virt-handler/virt_launcher.cil

```diff
@@ -8,7 +8,6 @@
     (allow process sysfs_t (file (write)))
     (allow process tmp_t (dir (write add_name open getattr setattr read link search remove_name reparent lock ioctl)))
     (allow process tmp_t (file (setattr open read write create getattr append ioctl lock)))
-    (allow process container_share_t (dir (write add_name)))
-    (allow process container_share_t (file (create setattr write )))
```

Comment:
I remember we've discussed this, so I'm just curious.
Don't these have any effect or do we re-label the directories?

Reply:
I was unable to find any directories labelled container_share_t. If that does occur in situations I'm unaware of, then the correct course of action is to relabel because container_share_t is never supposed to be write-able.